enterprise_access/apps/bffs/serializers.py

```diff
@@ -152,12 +152,24 @@ class EnterpriseCustomerSerializer(BaseBffSerializer):
     show_integration_warning = serializers.BooleanField()
 
 
+class EnterpriseCustomerUserSerializer(BaseBffSerializer):
+    """
+    Serializer for all linked enterprise customer users
+    """
+    id = serializers.IntegerField()
+    user_id = serializers.IntegerField()
+    enterprise_customer = EnterpriseCustomerSerializer()
+    active = serializers.BooleanField()
+
+
 class BaseResponseSerializer(BaseBffSerializer):
     """
     Serializer for base response.
     """
 
     enterprise_customer = EnterpriseCustomerSerializer(required=False, allow_null=True)
+    all_linked_enterprise_customer_users = EnterpriseCustomerUserSerializer(many=True, allow_empty=True, default=list)
+    should_update_active_enterprise_customer_user = serializers.BooleanField(default=False)
```

Comment:
FWIW, I do think `serializers.BooleanField` does default to `False` on its own, where passing `default=False` isn't needed ([source](https://www.geeksforgeeks.org/boolean-fields-in-serializers-django-rest-framework/)).

<img width="1027" alt="image" src="https://github.com/user-attachments/assets/1de158d7-bc97-4a4d-8798-1b589bf7574c" />

All tests pass without the `default=False`.